config/model_configs/prognostic_edmfx_soares_column.yml

```diff
@@ -0,0 +1,35 @@
+initial_condition: "Soares"
+surface_setup: "Soares"
+rayleigh_sponge: true
+turbconv: "prognostic_edmfx"
+implicit_diffusion: true
+implicit_sgs_advection: false
+approximate_linear_solve_iters: 2
+edmfx_upwinding: "first_order"
+edmfx_entr_model: "Generalized"
+edmfx_detr_model: "SmoothArea"
+edmfx_sgs_mass_flux: true
+edmfx_sgs_diffusive_flux: true
+edmfx_nh_pressure: true
+edmfx_filter: true
+prognostic_tke: false # should we set this as true or false?
+moist: "equil"
+config: "column"
+hyperdiff: false
+z_max: 3750
+z_elem: 75
+z_stretch: false
+perturb_initstate: false
+dt: "5secs"
+t_end: "8hours"
+dt_save_state_to_disk: "14hours"
```

Comment:
```suggestion
```
(14 hours is longer than t_end, so you can just remove this line. The default is Inf.)